api/v1alpha1/ndb_api_helpers.go

```diff
@@ -195,10 +245,25 @@ func GetOOBProfiles(ctx context.Context, ndbclient *ndbclient.NDBClient, dbType
 	profileMap[PROFILE_TYPE_SOFTWARE] = softwareProfiles[0]
 	profileMap[PROFILE_TYPE_NETWORK] = networkProfiles[0]
 	profileMap[PROFILE_TYPE_DATABASE_PARAMETER] = dbParamProfiles[0]
-
 	return
 }
 
+func GetProfileByType(profileType string, profiles Profiles) Profile {
+	defaultEmptyProfile := Profile{}
+	switch profileType {
+	case PROFILE_TYPE_COMPUTE:
+		return profiles.Compute
+	case PROFILE_TYPE_SOFTWARE:
+		return profiles.Software
+	case PROFILE_TYPE_NETWORK:
+		return profiles.Network
+	case PROFILE_TYPE_DATABASE_PARAMETER:
+		return profiles.DbParam
+	default:
+		return defaultEmptyProfile
+	}
+}
```

Comment:
This function is not used in the business logic, I only see it in one of the test files. How do you feel about placing it under the test files instead?
(Since it does not add any functionality/value in the ndb_api_helpers.go file)